scripts/release.py

```diff
@@ -236,7 +273,9 @@ def parse_next_version(version: Text) -> Text:
         return str(Version.coerce(get_current_version()).next_minor())
     elif version == "patch":
         return str(Version.coerce(get_current_version()).next_patch())
-    elif semantic_version.validate(version):
+    elif version == "alpha":
```

Comment:
optional: since we might soon want to cut an `rc`, can we easily make this more generic to also support these versions?

Reply:
good, so we would add support for `X.YrcN` from PEP spec